staking/types/validator.go

```diff
@@ -136,11 +136,14 @@ type ValidatorRPCEnchanced struct {
 	TotalDelegated       *big.Int                 `json:"total-delegation"`
 	CurrentlyInCommittee bool                     `json:"currently-in-committee"`
 	EPoSStatus           string                   `json:"epos-status"`
+	Lifetime             *AccumulatedOverLifetime `json:"lifetime"`
```

Comment:
Performance? or Uptime? Lifetime is bad naming.

Reply:
What’s better ? I’ll iterate with pangaea volunteers for good naming